src/models/ProjectUser/index.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import createProjectUser from './createProjectUser'
+export * from './createProjectUser'
+
+export type ProjectUser = {
+  id: number,
+  projectId: number,
+  userId: number,
+  role: string
```

Comment:
You could do: `role: ['OWNER', 'CONTRIBUTOR']` instead (we can change the actual values later)